frontend/src/shared/filter/components/type/filter-type-search.vue

```diff
@@ -76,7 +76,7 @@ watch(
   () => activeView.value,
   (newActiveView, oldActiveView) => {
     if (newActiveView.id !== oldActiveView.id) {
-      setModelValue('')
+      model.value = ''
```

Comment:
Just want to make sure this isn't breaking the previous fix we made a few time ago. Is search value still being cleared in the store after the tab has changed?

Reply:
yes it clears, just it doesn't trigger refetch